src/main/webapp/app/shared/util/utils.tsx

```diff
@@ -308,3 +309,21 @@ export function extractPositionFromSingleNucleotideAlteration(alteration: string
     return null;
   }
 }
+
+/* eslint-disable @typescript-eslint/prefer-regexp-exec */
```

Comment:
I've seen this multiple time. Is it worthy for us just disable this check?

Reply:
Yeah I would say so. Will look into why we use that check in the first place, but it just feels inconvenient sometimes.

Reply:
It's the default from eslint. Feel free to fine tone it in https://github.com/oncokb/oncokb-transcript/blob/rc/.eslintrc.json